core/whistle_media-1.0.0/src/wh_media_recording.erl

```diff
@@ -355,7 +366,7 @@ handle_cast({'gen_listener',{'is_consuming', 'true'}}, #state{record_on_answer='
                                                              }=State) ->
     start_recording(Call, MediaName, TimeLimit, <<"wh_media_recording">>, SampleRate, RecordMinSec),
     lager:debug("started the recording"),
-    {'noreply', State};
+    {'noreply', State#state{is_recording='true'}};
```

Comment:
Actually, this just popped into my thought after I merged - we've only send the command to start recording here but haven't yet received the `RECORD_START` event, so why this setting of `is_recording` to `true`? Shouldn't the server wait for `RECORD_START`?